meta-resin-common/recipes-containers/balena/balena/balena-host.service

```diff
@@ -3,7 +3,7 @@ Description=Balena Application Container Engine (host)
 Documentation=https://www.balena.io/docs/getting-started
```

Comment:
It is not clear why we want this dependency from the git log.

Reply:
I'll elaborate the gitlog a bit more. No point in running balena as if rollback-altboot is triggering, it will run hooks and reboot.

Reply:
Added some more text in the commit message